### PR TITLE
[test](mtmv) Fix regression test unstable

### PR DESCRIPTION
**Pull request by seawinde**
## Proposed changes

1. Optimized `waitForRollUpJob` method in `Suite.groovy` to make sure roll up is build succesfully before rewirte.
2. Modify expect rewritten result in` agg_sync_mv.groovy`

### CODE REVIEW

**Comment by seawinde** [comment]
run buildall

File: regression-test/suites/nereids_rules_p0/mv/agg_optimize_when_uniform/agg_optimize_when_uniform.groovy [hunk 1]
```diff
@@ -115,10 +115,19 @@ suite("agg_optimize_when_uniform") {
...
     (1, 1, 'o', 10.5, '2023-12-08', 'a', 'b', 1, 'yy'),
+    (1, 1, 'o', 10.5, '2023-12-08', 'a', 'b', 1, 'yy'),
+    (2, 1, 'o', 11.5, '2023-12-09', 'a', 'b', 1, 'yy'),
+    (2, 1, 'o', 11.5, '2023-12-09', 'a', 'b', 1, 'yy'),
     (2, 1, 'o', 11.5, '2023-12-09', 'a', 'b', 1, 'yy'),
     (3, 1, 'o', 12.5, '2023-12-10', 'a', 'b', 1, 'yy'),
     (3, 1, 'o', 33.5, '2023-12-10', 'a', 'b', 1, 'yy'),
+    (3, 1, 'o', 33.5, '2023-12-10', 'a', 'b', 1, 'yy'),
+    (3, 1, 'o', 33.5, '2023-12-10', 'a', 'b', 1, 'yy'),
     (4, 2, 'o', 43.2, '2023-12-11', 'c','d',2, 'mm'),
```
Comment (morrySnow) [inline regression-test/suites/nereids_rules_p0/mv/agg_optimize_when_uniform/agg_optimize_when_uniform.groovy:126]: why update data?

Comment (seawinde) [reply regression-test/suites/nereids_rules_p0/mv/agg_optimize_when_uniform/agg_optimize_when_uniform.groovy:126]: Becaus the case is not chosen by CBO stably， add more data to make cbo chose mv easier.
but there is a fix pr https://github.com/apache/doris/pull/40529 to fix this. Maybe we doesn't need to update data.

**Comment by seawinde** [comment]
run buildall

**Comment by seawinde** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**